themes/default/footer.inc

```diff
@@ -1,3 +1 @@
-<p style="text-align: right">
- Created by <a href="http://steve.org.uk/Software/chronicle">Chronicle v<!-- tmpl_var name='chronicle_release' --></a>
-</p>
+<p style="text-align: right">Created by <a href="http://steve.org.uk/Software/chronicle">Chronicle v[% chronicle_release %]</a></p>
```

Comment:
I notice the markup has changed from `<!-- .... -->` to `[% --- %]` here.

I expect that this should be reverted such that the default footer works with the default engine, HTML::Template.